templates/sounds/sound_edit.html

```diff
@@ -279,6 +279,14 @@ <h3>Delete</h3>
         $("#{{geotag_form.zoom.auto_id}}").val(Math.floor(zoom));
     }
 
+    function centerMapToLatLonZoomFields() {
+        var lat = $("#{{geotag_form.lon.auto_id}}").val();
+        var lon = $("#{{geotag_form.lat.auto_id}}").val();
+        var zoom = $("#{{geotag_form.zoom.auto_id}}").val();
+        window.map.setZoom(parseInt(zoom, 10) - 1);
```

Comment:
What happens if these values aren't numbers?

Reply:
The `<input>` type is `number` and the browser adds some validation (if not a number, undoes the change). If this did not happen I guess there would be a javascript error (unseen by the user) and the map would not update. That would still make sense as the input value would be invalid. To be honest I'm not too much worried by this behaviour and I guess we can trust the input type thing.